scripts/cli2md.rb

```diff
@@ -0,0 +1,21 @@
+ARGF.each_with_index do |line, line_num|
+    # Headings 
+    if /^([A-z]*):/ =~ line
```

Comment:
Might be better to just use the "word character" selector here; `A-z` includes a bunch of other characters which aren't alphabetical:

```suggestion
    if /^(\w*):/ =~ line
```